internal/storage/storagelocation_test.go

```diff
@@ -42,45 +42,56 @@ func TestIsReadyToValidate(t *testing.T) {
 		ready bool
 	}{
 		{
-			name:                   "don't validate, since frequency is set to zero",
+			name:                   "should return true when validation frequency is zero and lastValidationTime is nil",
```

Comment:
I chose to use the variable names as they were more descriptive and the reason for test failure can be easily narrowed down.
The reason for this change is that the test was incorrect. We want this method to return `true` in this test but the previously expected result was `false`. Also, the diff seems misaligned.

Reply:
Also, preferring the "should" naming convention as it is more direct in what should happen, and when the test fails we will know exactly what should have happened.

Reply:
Marking this as resolved.
@carlisia Please unresolve it if you feel strongly that this needs to be changed.

Reply:
I really would like to have the changes I mentioned, mainly, not to introduce a new way to name tests (we consistently use an active voice instead of "should"), and avoid using variable names in the test name that we'd have to change if we rename the variable. Thank you! This is the only change I'd like to see for this and then it's ready to merge.

Reply:
Done!